src/v/crypto/tests/BUILD

```diff
@@ -84,3 +86,133 @@ redpanda_cc_gtest(
         "@googletest//:gtest",
     ],
 )
+
+expand_template(
+    name = "openssl_conf",
+    out = "openssl_conf.cnf",
+    substitutions = {
+        "@REDPANDA_DEPS_INSTALL_DIR@": "src/v/crypto/tests",
+    },
+    template = "test_openssl_conf.cnf.in",
+)
+
+select_file(
+    name = "fipsmodule_module",
+    srcs = "@openssl-fips",
+    subpath = "lib/ossl-modules/fips.so",
+)
```

Comment:
Can we move this to the openssl-fips BUILD file?

Reply:
done